tests/common/devices/sonic.py

```diff
@@ -1340,3 +1340,69 @@ def get_up_ip_ports(self):
             except KeyError:
                 pass
         return up_ip_ports
+
+    def get_rsyslog_ipv4(self):
```

Comment:
These changes should already be in through other PRs. Can you update the branch and push?

Reply:
that' right, that's why I didn't assign anyone yet.. It also needs a merge.